Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -228,11 +228,17 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
 
     @objc public let attribution: Attribution
 
+    @available(*, deprecated, message: "Use purchasesAreCompletedBy instead.")
     @objc public var finishTransactions: Bool {
         get { self.systemInfo.finishTransactions }
         set { self.systemInfo.finishTransactions = newValue }
     }
 
+    @objc public var purchasesAreCompletedBy: PurchasesAreCompletedBy {
+        get { self.systemInfo.finishTransactions ? .revenueCat : .myApp }
+        set { self.systemInfo.finishTransactions = (newValue == .revenueCat ? true : false) }
```

Comment:
I would maybe add an extension into the enum called `finishTransactions` for this where `.revenueCat` return `true` and `myApp` returns `false`?

That way this logic is contained in the enum and can be used later directly somewhere else when we refactor this variable away?

Reply:
2 idiots.. as the Germans would apparently say 😅 
https://github.com/RevenueCat/purchases-android/blob/main/purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesAreCompletedBy.kt#L27

Reply:
lol folks reading this thread without seeing the slack context would probably think our PRs are toxic 😂

Reply:
We should open source our slack! (jk) 
  
(I changed it to a similar spirited line, but one that needs less context 😛)